mule-packager/src/main/java/org/mule/tools/api/verifier/policy/PolicyMuleArtifactJsonVerifier.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.api.verifier.policy;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.lang.String.format;
+import static org.mule.tools.api.packager.packaging.Classifier.MULE_POLICY;
+
+import org.mule.runtime.api.deployment.meta.MuleApplicationModel;
+import org.mule.runtime.api.deployment.persistence.MuleApplicationModelJsonSerializer;
+import org.mule.tools.api.exception.ValidationException;
+import org.mule.tools.api.packager.ProjectInformation;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Map;
+
+import org.apache.commons.io.FileUtils;
+
+class PolicyMuleArtifactJsonVerifier {
+
+  private static final String GROUP_ID = "groupId";
+  private static final String ARTIFACT_ID = "artifactId";
+  private static final String VERSION = "version";
+  private static final String CLASSIFIER = "classifier";
+  private static final String TYPE = "type";
+  private static final String JAR_TYPE = "jar";
+  private static final String EXPORTED_PACKAGES = "exportedPackages";
+  private static final String EXPORTED_RESOURCES = "exportedResources";
+  private static final String CONFIGS = "configs";
+  public static final String TEMPLATE_XML = "template.xml";
+
+  public static void validate(ProjectInformation projectInformation, File file) throws ValidationException {
```

Comment:
I would remove the static, find the common behaviour, and create the abstract class/interface, if it clashes with smth that already exists, then this is not the correct place to put it. Otherwise, you've found a new concept in the domain of what this repo models, and it'll probably be of use for future cases

Reply:
you already said this in other comment, already fixed.